tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -115,6 +116,18 @@ Search and Verify GPU Items Appears In Resources Page
     Search Items In Resources Section   GPU
     Check GPU Resources
 
+Verify Notebook images are building is not shown when no images are building
+    [Documentation]     Verifies that RHODS Dashbaord doesn't contain Notebook Images are building Notification , if no build is running
+    [Tags]    Sanity
+    ...       ODS-307
+    ${builds_data}=  Oc Get  kind=Build  namespace=redhat-ods-applications
+    FOR    ${build_data}    IN    @{builds_data}
+        Wait Until Build Status Is    namespace=redhat-ods-applications    build_name=${build_data['metadata']['name']}  expected_status=Complete
+    END
```

Comment:
as per style guide, it'd be better to avoid Loop/IF statementes in the TC. Could you move this to a kw?

Reply:
Updated.